multiqc/modules/sickle/sickle.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+
+
+""" MultiQC module to parse output from sickle """
+
+from multiqc import config
+from multiqc.modules.base_module import BaseMultiqcModule
+import logging
+import re
+from multiqc.plots import bargraph
+from collections import OrderedDict
+
+log = logging.getLogger(__name__)
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        super(MultiqcModule, self).__init__(name='sickle', anchor='sickle',
+                                            href="https://github.com/najoshi/sickle",
+                                            info="A windowed adaptive trimming tool for FASTQ files using quality.")
+
+        self.sickle_data = dict()
+        #parse list of log files
+        for f in self.find_log_files('sickle'):
+            self.sickle_data[f['s_name']] = self.parse_logs(f['f'])
+            self.add_data_source(f)
+        #no file found
+        if len(self.sickle_data) == 0:
+            raise UserWarning
+        self.sickle_data = self.ignore_samples(self.sickle_data)
```

Comment:
This needs to go before the above check, as if samples are ignored then we might end up with 0..